client/gutenberg/editor/calypsoify-iframe.jsx

```diff
@@ -181,6 +191,20 @@ class CalypsoifyIframe extends Component {
 		this.setState( { isMediaModalVisible: false } );
 	};
 
+	closeClassicBlockMediaModal = () => this.setState( { isClassicBlockMediaModalVisible: false } );
+
+	insertClassicBlockMedia = markup => {
+		if ( this.iframePort ) {
```

Comment:
Note that you can also send a specific `MessageChannel` port with the `classicBlockOpenMediaModal` message we can use for communicating back to WP.com.

That way we don't use the general-purpose `iframePort` channel in favor of a more isolated channel. We already do that for the Media Modal:

https://github.com/Automattic/wp-calypso/blob/f159e1638a29c158aa575641e42f918e1b2f8667/client/gutenberg/editor/calypsoify-iframe.jsx#L101-L106

Reply:
I'm not sure how to do this server side as the code that sends the message is not in the iframe bridge, but in a TinyMCE plugin. 🤔

Reply:
Oh right, we don't have the logic available in the TinyMCE plugin. I left a comment in the diff wondering if we should expose part of the iframe bridge logic, so we can use it on the plugin.